scene/resources/animation.cpp

```diff
@@ -1469,7 +1469,7 @@ int Animation::_find(const Vector<K> &p_keys, float p_time) const {
 
 		middle = (low + high) / 2;
 
-		if (p_time == keys[middle].time) { //match
+		if (Math::absf(p_time - keys[middle].time) < CMP_EPSILON) { //match
```

Comment:
You can use `Math::abs` directly, I don't think `absf` and `absd` are meant to be used.

Reply:
That's probably by oversight, `Math::abs` already calls `Math::absf` or `Math::absd` internally, so unless you want to make 100% that it will be float and never double, `Math::abs` should be used.
```
$ rg "Math::abs[^df]" | wc -l
145
$ rg "Math::abs[df]" | wc -l
14
```

Reply:
Allright, fixed.